cypress-custom/integration/fee.test.ts

```diff
@@ -21,42 +22,47 @@ describe('Fetch and persist fee', () => {
   beforeEach(() => {
     cy.visit('/swap')
 
-    // Alias localStorage
     cy.window()
       .then(window => window.localStorage)
       .as('localStorage')
   })
 
   it('Persisted when selecting a token', () => {
-    const TOKEN = 'ETH'
-    const NETWORK = ChainId.RINKEBY.toString()
-    // GIVEN: A fee for a token is not in the local storage
-    // WHEN: When the user select this token
+    const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
+    const Rinkeby = ChainId.RINKEBY.toString()
 
-    // Clear localStorage
-    cy.get<Storage>('@localStorage').then(storage => storage.clear())
+    // GIVEN: Clean local storage
+    cy.clearLocalStorage()
 
-    // THEN: The fee is persisted in the local storage (redux_localstorage_simple_fee)
-    cy.get<Storage>('@localStorage')
-      .its('redux_localstorage_simple_fee')
-      .should($feeStorage => {
-        // we need to parse JSON
-        const feeStorage = JSON.parse($feeStorage)
-        const feeQuote = feeStorage[NETWORK][TOKEN]
-        expect(feeStorage).to.have.property(NETWORK)
-        expect(feeStorage[NETWORK]).to.have.property(TOKEN)
-        expect(feeQuote.token).to.equal(TOKEN)
-        expect(feeQuote.fee).to.have.property('minimalFee')
-        expect(feeQuote.fee).to.have.property('feeRatio')
-        expect(feeQuote.fee).to.have.property('expirationDate')
+    // WHEN: Select DAI token
+    cy.swapSelectInput(DAI)
+
+    // THEN: The fee for DAI is fetched
+    cy.window()
```

Comment:
don't need this with the alias in `beforeEach`

use instead:
`cy.get('@localStorage')`

Reply:
I find the alias make it more difficult to understand imo, although it was a fair use of them. 
My point is that u rely on something declared in the beforeEach so it feels less self-contained in my opinion. I my approach, the code is equivalent, and if I don't know what `_getLocalStorage()` does, I ctrl-click and go to the implementation. (see https://github.com/gnosis/gp-swap-ui/pull/123#discussion_r566772019)

With alias u need to find a ".as('aliasName')" somewhere either in the test or in the before, or in the beforeEach. Typescript will not now where to look or if it's right or wrong if you misspell it. 

As I said, u used as Cypress mentions it in the docs, and for the reason they created it, this is just my personal opinion. 
Happy to change it back.

Reply:
as I mentioned [here](https://github.com/gnosis/gp-swap-ui/pull/123#pullrequestreview-579391804) I don't think it's harder to read but I do like the function trace in VS with that approach. 

the type for `cy.get` btw is passable e.g `cy.get<Storage>('@localStorage')`